app/services/cfe_service.rb

```diff
@@ -0,0 +1,156 @@
+  end
+
+  def save_vehicle_finance_data(cfe_estimate_id, cfe_session_data)
+    value_form = Flow::Vehicle::ValueHandler.model(cfe_session_data)
+    return unless value_form.vehicle_value.present?
+
+    age_form = Flow::Vehicle::AgeHandler.model(cfe_session_data)
+    date_of_purchase = age_form.vehicle_over_3_years_ago ? 4.years.ago.to_date : 2.years.ago.to_date
+    cfe_connection.create_vehicle cfe_estimate_id,
+                                  date_of_purchase:,
+                                  value: value_form.vehicle_value,
+                                  loan_amount_outstanding: form.vehicle_pcp ? form.vehicle_finance.presence : 0,
+                                  in_regular_use: value_form.vehicle_in_regular_use
+  end
+
+  def save_assets_data(cfe_estimate_id, cfe_session_data)
+    form = Flow::AssetHandler.model(cfe_session_data)
+    savings = [form.savings].compact
+    investments = [form.investments].compact
+    cfe_connection.create_capitals cfe_estimate_id, savings, investments
+
+    if form.assets.include?("property")
+      property_entry_form = Flow::PropertyEntryHandler.model(cfe_session_data)
+      main_home = {
+        value: property_entry_form.house_value,
+        outstanding_mortgage: property_entry_form.mortgage,
+        percentage_owned: property_entry_form.percentage_owned,
+      }
+      second_property = {
+        value: form.property_value,
+        outstanding_mortgage: form.property_mortgage,
+        percentage_owned: form.property_percentage_owned,
+      }
+      cfe_connection.create_properties(cfe_estimate_id, main_home, second_property)
+    end
+  end
+
+  def save_outgoings_data(cfe_estimate_id, cfe_session_data)
+    form = Flow::OutgoingsHandler.model(cfe_session_data)
+    income_form = Flow::MonthlyIncomeHandler.model(cfe_session_data)
+
+    cfe_connection.create_regular_payments(cfe_estimate_id, income_form, form)
+  end
+
+  def save_property_entry_data(cfe_estimate_id, model)
+    return unless model.house_value.present?
+
+    main_home = {
+      value: model.house_value,
+      outstanding_mortgage: (model.mortgage.presence if cfe_session_data["property_owned"] == "with_mortgage") || 0,
+      percentage_owned: model.percentage_owned
+    }
+    cfe_connection.create_properties(cfe_estimate_id, main_home, nil)
+  end
+
+  def create_applicant(cfe_estimate_id, model)
+    cfe_connection.create_applicant cfe_estimate_id,
+                                    date_of_birth: model.over_60 ? 70.years.ago.to_date : 50.years.ago.to_date,
+                                    receives_qualifying_benefit: model.passporting
+  end
+
+  # CFE doesn't understand about annual salary or 'total income in the last 3 months',
+  # so for both those use cases we must convert the figures we have into what they would
+  # be if they were paid as regular monthly income
+  def multiplier(form)
+    case form.frequency
+    when "annually"
+      1.0 / 12
+    when "total"
+      1.0 / 3
+    else
```

Comment:
The scope of these methods (in particular) make me think that everything in here could be a class method rather than an instance method.